server/controllers/Auth.js

```diff
@@ -32,7 +27,7 @@ class Auth {
     };
     const user = await createSocialUsers(data);
     delete user.password;
-    serverResponse(response, 200, user);
+    response.redirect(`${process.env.CLIENT_URL}?token=${user.token}`);
```

Comment:
change to `process.env.CLIENT_URL}/${user.token}`

Reply:
I've done this and on the frontend, it includes the token as part of the route and tries to access that route. Because there is no such route on the frontend, the request fails. 
<img width="1440" alt="Screenshot 2019-09-04 at 9 51 47 PM" src="https://user-images.githubusercontent.com/42539315/64290871-d6da0a80-cf5e-11e9-9f96-9be337423886.png">

Reply:
So do you have a route like the one you are pointing to?